collector.go

```diff
@@ -45,15 +45,17 @@ var (
 			"disabled":            {GAUGE, "disabled", 1, "1 if this database is currently disabled, else 0"},
 		},
 		"stats": {
-			"database":          {LABEL, "N/A", 1, "N/A"},
-			"total_query_count": {COUNTER, "queries_pooled_total", 1, "Total number of SQL queries pooled"},
-			"total_query_time":  {COUNTER, "queries_duration_seconds_total", 1e-6, "Total number of seconds spent by pgbouncer when actively connected to PostgreSQL, executing queries"},
-			"total_received":    {COUNTER, "received_bytes_total", 1, "Total volume in bytes of network traffic received by pgbouncer, shown as bytes"},
-			"total_requests":    {COUNTER, "queries_total", 1, "Total number of SQL requests pooled by pgbouncer, shown as requests"},
-			"total_sent":        {COUNTER, "sent_bytes_total", 1, "Total volume in bytes of network traffic sent by pgbouncer, shown as bytes"},
-			"total_wait_time":   {COUNTER, "client_wait_seconds_total", 1e-6, "Time spent by clients waiting for a server in seconds"},
-			"total_xact_count":  {COUNTER, "sql_transactions_pooled_total", 1, "Total number of SQL transactions pooled"},
-			"total_xact_time":   {COUNTER, "server_in_transaction_seconds_total", 1e-6, "Total number of seconds spent by pgbouncer when connected to PostgreSQL in a transaction, either idle in transaction or executing queries"},
+			"database":                      {LABEL, "N/A", 1, "N/A"},
+			"avg_wait_time":                 {GAUGE, "avg_wait_seconds", 1e-6, "Average time spent by clients waiting for a server, in seconds, for the last stats period"},
```

Comment:
We explicitly don't include `avg_wait_time` since it's a lossy gauge and can be computed from `total_wait_time`.